pyfar/constants/constants.py

```diff
@@ -433,3 +434,149 @@ def density_of_air(
 
     # Equation 6.71
     return P / (R_a * (temperature + 273.15)) * (1+C)/(1+alpha*C)
+
+def fractional_octave_frequencies_nominal(num_fractions=1,
+                                          frequency_range=(20, 20e3)):
+    """Return the nominal center frequencies for octave-band and
+    one-third-octave-band filters.
+
+    Nominal center frequencies, as specified in the IEC 61260-1:2014 standard
+    [#]_, are standardized values that approximate the exact center
+    frequencies. In this function they are defined only for the range from
+    10 Hz to 20 kHz.
+
+    Parameters
+    ----------
+    num_fractions : 1, 3
+        The number of octave fractions. ``1`` returns octave center
+        frequencies, ``3`` returns third octave center frequencies.
+    frequency_range : array, tuple
+        The lower and upper frequency limits, the default is
+        ``(20, 20e3)``.
+        ``(20, 20e3)`` is in accordance with IEC 61260-1
+        ``(10, 20e3)`` is in accordance with IEC 61672-1 [#]_
+
+    Returns
+    -------
+    nominal : numpy.ndarray of float
+        The nominal center frequencies.
+
+    References
+    ----------
+    .. [#] International Electrotechnical Commission, "IEC 61260-1:2014 -
+        Electroacoustics - Octave-band and fractional-octave-band filters -
+        Part 1: Specifications", IEC, 2014.
+
+    .. [#] International Electrotechnical Commission,
+        "IEC 61672-1:2013 - Electroacoustics - Sound level meters - Part 1:
+        Specifications", IEC, 2013.
+    """
+    G = 10**(3/10)
+    f_lims = np.asarray(frequency_range)
+    if f_lims.size != 2:
+        raise ValueError(
+            "You need to specify a lower and upper limit frequency.")
+    if f_lims[0] > f_lims[1]:
+        raise ValueError(
+            "The second frequency needs to be higher than the first.")
+    #
+    if num_fractions == 1:
+            if (f_lims[0] > 15.8*G**(-1/2)) and ( f_lims[1] <
+                                                 15848.93192*G**(1/2)):
+                nominal = np.array([
+                    16, 31.5, 63, 125, 250, 500, 1e3,
+                    2e3, 4e3, 8e3, 16e3], dtype=float)
+            else:
+                warnings.warn(
```

Comment:
this warning should also be checked by a test. See test_deprecations.py for an example